src/packages/vcPing.py

```diff
@@ -39,7 +46,8 @@ def is_vc(self, channel: discord.abc.GuildChannel) -> bool:
 		return True
 
 	# returns the role if it exists otherwise, create the role.
-	async def compute_role_if_absent(self, roleName: str, guild: discord.Guild, suffix=None) -> discord.Role:
+	@lru_cache(1)
```

Comment:
👀

Reply:
cache go brrrrrr

Reply:
such a dumb fix tho lol, only caches the last computed thing but it might just fix a weird bug I can't recreate